src/python/foglamp_daemon.py

```diff
@@ -21,9 +23,7 @@ def do_something(logf):
     logger.addHandler(fh)
     logger.setLevel(logging.DEBUG)
 
-    # set DB config
-    Configurator().initialize_dbconfig()
-    CoAPServer.start()
+    start.start()
```

Comment:
Agree but I'm not sure I'm liking using module def's without classes. What do you think?